### PR TITLE
Use chandra limits

### PR DESCRIPTION
**Pull request by jzuhone**
## Description
This version of `acis_thermal_check` hands violations checking and plotting of limits over completely to the `chandra_limits` package. A review of the most important changes is as follows:

* As noted above, the violations checking code has been ripped out. The code that is now in `chandra_limits` is largely copied and refactored over from the code originally in `acis_thermal_check`. 
* For the ACIS FP model, instead of reporting a table of "hot ACIS" observations, there is a new page linked to from the main page reporting a table of all observations and their properties in the schedule. 
* The planning limit lines are now piece-wise constant, especially in the the case of the ACIS FP. Yellow limit lines remain the same. 
* In the ACIS FP plots, the obsids are now only colored red and green depending on whether they are ACIS-I or ACIS-S. 
* The validation limits code has been simplified as there was a lot of duplication. This largely occurred in commit d591e91588a30ac9754b1bebadb20e15544eb6dd.

Practical effects of these changes:
* Plots look somewhat different because of the change in planning limits, especially the ACIS FP plot. 
* ACIS FP data quality violations are no longer reported during the taking of a bias.
* ACIS FP -105 C observations are now supported. One of the plots has had its upper limit set to -103 C to accommodate this. 

## Interface impacts

The table reporting violations on the webpages has changed slightly, but is human-read. Plots show limit lines as a function of time instead of every limit line across the whole plot; also human-read.

### Unit tests

- [ ] No unit tests
- [x] Mac
- [ ] Linux
- [ ] Windows

All tests which do not depend on violations checking pass without any answer updates as expected. Some answer changes are necessary for the FP and a couple of other models. All of these answer changes were hand-checked and they were found to be correct based on the code changes.

Independent check of unit tests by [REVIEWER NAME]
- [ ] [PLATFORM]:

### Functional tests

Ran the code in this PR at weekly load reviews for several weeks and results were checked for consistency with flight code; any differences are explained by changes to the violations checking changes.

### CODE REVIEW

File: acis_thermal_check/tests/bep_pcb/answers/JUL2919A_viol.json
Comment (jazan12) [inline acis_thermal_check/tests/bep_pcb/answers/JUL2919A_viol.json:1]: Just want to confirm the intentionality of having syntax of odb.caution for the limits for the official models vs the safety.caution for the "informal" models.

File: acis_thermal_check/main.py [hunk 1]
```diff
@@ -1393,6 +1286,23 @@ def write_index_rst(self, outdir, context):
         # Render the template and write it to a file
         with open(outfile, "w") as fout:
             fout.write(template.render(**context))
+        if self.msid == "fptemp":
+            # For the ACIS FP limit we write out an obsid table
+            context["acis_obs"] = self.acis_and_ecs_obs
+            template_path = (
+                TASK_DATA / "acis_thermal_check/templates/obsid_template.rst"
```
Comment (jazan12) [inline acis_thermal_check/main.py:1293]: I don't see where TASK_DATA is defined.

Comment (jeanconn) [reply acis_thermal_check/main.py:1293]: In passing - I think it is imported from utils here https://github.com/acisops/acis_thermal_check/blob/4f7c496bb51f327b16a75066896d5c5e5dfbe55d/acis_thermal_check/main.py#L25

File: acis_thermal_check/tests/bep_pcb/answers/JUL2919A_viol.json
Comment (jzuhone) [reply acis_thermal_check/tests/bep_pcb/answers/JUL2919A_viol.json:1]: It's not so much "official" vs "informal" as what limits are stored in the ODB (related to engineering telemetry) vs the ones related to DEA HKP telemetry

**Comment by jazan12** [comment]
Thanks

**Comment by jazan12** [comment]
Good, so long as the syntax was as intended

**Review by jazan12 — APPROVED** [review]
All my comments were addressed.  Looks good!

**Review by Gregg140 — APPROVED** [review]

**Pull request merged**